src/util.ts

```diff
@@ -71,27 +72,58 @@ export const goBuiltinTypes: Set<string> = new Set<string>([
 	'uintptr'
 ]);
 
-export interface SemVersion {
-	major: number;
-	minor: number;
-}
+export class GoVersion {
+	sv: semver.SemVer;
+	commit: string;
+	isDevel: boolean;
+
+	constructor(version: string) {
+		const matchesRelease = /go version go(\d.\d+).*/.exec(version);
+		const matchesDevel = /go version devel \+(.\d+).*/.exec(version);
+		if (matchesRelease) {
+			this.sv = semver.coerce(matchesRelease[0]);
+		} else if (matchesDevel) {
+			this.isDevel = true;
+			this.commit = matchesDevel[0];
+		}
+		/* __GDPR__
+            "getGoVersion" : {
+	  			"version" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
+   			}
+ 		*/
+		sendTelemetryEvent('getGoVersion', { version: this.format() });
+	}
 
-let goVersion: SemVersion = null;
-let vendorSupport: boolean = null;
-let telemtryReporter: TelemetryReporter;
-let toolsGopath: string;
+	format(): string {
+		if (this.sv) {
+			return `${this.sv.major}.${this.sv.minor}`;
```

Comment:
Apparently there is a property "raw" that would give the version in the string format. Have you tried that?

Reply:
Used the `.format()` function instead. Thanks!